pandas/core/series.py

```diff
@@ -4355,14 +4355,16 @@ def align(
 
     def rename(
         self,
-        index=None,
+        mapper=None,
         *,
+        index=None,
```

Comment:
umm, you are adding args?

Reply:
Yes, but it's just for compatibility with the signature in `generic.py`, from a user perspective everything should be the same

Reply:
Yes - the idea is that now it's called `mapper`, but that if a user passes `index`, then it'll still work as before (for backwards compatibility).
And if someone passes both `mapper` and `index`, it'll raise an error